Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java

```diff
@@ -220,13 +222,34 @@ private void updateMP(int maxMP) {
                 e.printStackTrace();
             }
         }
+
         Text player2MPText = new Text(player2.getCurrentMP() + "/" + maxMP);
         player2MPText.setX(SCREEN_WIDTH - SCALE * (335 + maxMP * 40));
         player2MPText.setY(SCALE * (180 - maxMP * 2));
```

Comment:
These` SCALE * (180 - maxMP * 2) `and `SCALE * (300 + maxMP * 40)` seem to pop up in several places. Would it make sense to introduce variable for these things, to reduce duplication?

Reply:
Maybe? I think we'd need to simplify the math first to make it easier

Reply:
I think first step could be just introduce constant for these equations. 'cause I have no idea why they are like. 😅

Reply:
all of the capitalized variables are constants already ://///

Reply:
Okay yeah, I took a bit clloser look, and it's not that easy. Whoops.  There are still some constants that could be extracted, but no biggie for now. 👍

Reply:
welcome to our spaghetti filled world :')